docs/user/dashboard/dashboard.asciidoc

```diff
@@ -182,32 +267,51 @@ dashboard.
 [role="screenshot"]
 image:images/clone_panel.gif[clone panel]
 +
-{kib} automatically saves the cloned panel in *Visualize*. To access the cloned panel, open the main menu, then click *Visualize*.
+To access the cloned panel, open the main menu, then click *Visualize Library*.
```

Comment:
Might need @ThomThomson to fact check me on this but if the panel is by-value and you clone it, it won't save it to the visualize library.

Reply:
Good catch, this is true! The clone of a by value visualization is still by value, and the clone of a by reference visualization is a newly created saved object. Definitely good to make that distinction clear.